tests/test_inner.py

```diff
@@ -526,6 +609,19 @@ class Exit(ExitPlugin):
      False,  # does not log error
     ),
 
+    # No such plugin, buildstep, not required
+    ({'buildstep_plugins': [{'name': 'no plugin',
+                             'args': {},
+                             'required': False},
+                            {'name': 'buildstep_watched',
+                             'args': {
+                                 'watcher': Watcher(),
+                             }
+                            }]},
+     False,  # not fatal
+     False,  # does not log error
+    ),
+
```

Comment:
Could we add a test-case here for when there are multiple buildstep_plugins which would succeed, to verify that only the first one is called?